docs/root/configuration/listeners/udp_filters/dns_filter.rst

```diff
@@ -73,19 +75,51 @@ Example Configuration
             - name: "www.domain4.com"
               endpoint:
                 cluster_name: cluster_0
+            - name: "voip.domain5.com"
```

Comment:
I would've expected this to just be name: "voip.domain5.com", with endpoint that says cluster_name: "voip_cluster_0" - which has the port, IP, priority, weight etc as part of the LB settings

Reply:
Right, I will circle back and see what can be changed here.  I was thinking purely in a DNS server context.